guides/metrics.md

```diff
@@ -0,0 +1,6 @@
+
+## Defalut metrics to track
+
+- For each API endpoint:
+    - sum of requests
+    - count of requests with label of http response code
```

Comment:
could you also add other metrics you plan to add here in this list? it's ok if they are not yet implemented

Reply:
Sure, will do.